package.json

```diff
@@ -27,10 +27,10 @@
     "execspawn": "^1.0.1",
     "ghreleases": "^3.0.2",
     "github-from-package": "0.0.0",
-    "glob": "^7.2.3",
+    "glob": "^10.3.10",
     "minimist": "^1.2.8",
     "napi-build-utils": "^1.0.2",
-    "node-abi": "^3.47.0",
+    "node-abi": "^3.51.0",
```

Comment:
This is the change we care about.

https://github.com/electron/node-abi/commit/85418439eef4e47cb53e180447caa69da2f00e97